project/scalatest.scala

```diff
@@ -133,6 +133,7 @@ object ScalatestBuild extends BuildCommons with DottyBuild with NativeBuild with
           </developer>
         </developers>
       ),
+    usePgpKeyHex("4b5961ebc147a7ca19ba39dd905fdd07"),
```

Comment:
@ohze May I know what this is used for?  Can it be checked in accidentally I wonder?